bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java

```diff
@@ -3170,13 +3114,10 @@ private void setVisibleColumnIds_FromMenu(final MenuItem[] menuItems) {
 
          final Object itemData = menuItem.getData();
 
-         if (itemData instanceof ColumnDefinition) {
+         if (itemData instanceof final ColumnDefinition colDef) {
 
             final boolean isChecked = menuItem.getSelection();
 
-            // data in the table item contains the input items for the viewer
```

Comment:
This comment is removed

Reply:
Fixed in this [PR](https://github.com/mytourbook/mytourbook/pull/1326)